src/lib/query.ts

```diff
@@ -4,11 +4,13 @@ import { BaseQueryParameters, QueryOperation, QueryOperator, TaxonomyQueryOperat
 export class Query extends BaseQuery {
   private _contentTypeUid?: string;
 
-  constructor(client: AxiosInstance, uid: string, queryObj?: { [key: string]: any }) {
+  constructor(client: AxiosInstance, params: { [key: string]: any }, queryParams: { [key: string]: string | boolean | number }, uid?: string, queryObj?: { [key: string]: any }) {
```

Comment:
@harshithad0703 add this type in a seperate file and use the type here instead of defining it everytime.